app/src/main/java/com/greenfox/gitinder/activity/MessagesActivity.java

```diff
@@ -0,0 +1,73 @@
+package com.greenfox.gitinder.activity;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.ImageButton;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.greenfox.gitinder.R;
+import com.greenfox.gitinder.adapter.MessageAdapter;
+import com.greenfox.gitinder.model.Message;
+import com.greenfox.gitinder.model.MessageWrapper;
+import com.squareup.picasso.Picasso;
+
+import java.util.List;
+
+public class MessagesActivity extends AppCompatActivity {
+
+    TextView usernameText;
+    ImageView userPic;
+    MessageAdapter messageAdapter;
+    EditText editText;
+    ImageButton sendButton;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_messages);
+
+        usernameText = findViewById(R.id.messages_activity_username);
+        userPic = findViewById(R.id.messages_activity_picture);
+        sendButton = findViewById(R.id.messages_activity_send_button);
+        editText = findViewById(R.id.messages_activity_edit_text);
+
+        usernameText.setText(getIntent().getStringExtra("profileUsername"));
```

Comment:
@kolekd you could pass the whole mathc object from the adapter that contains everything (username, pic, messages)